pkgs/by-name/sn/snowflake-cli/package.nix

```diff
@@ -0,0 +1,106 @@
+{
+  lib,
+  stdenv,
+  darwin,
+  python3Packages,
+  fetchFromGitHub,
+  installShellFiles,
+  fetchPypi,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "snowflake-cli";
+  version = "3.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "snowflakedb";
+    repo = "snowflake-cli";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-QBYdzKKkakgwCD12bN7CerjfvMnxvt2cJ0NqTIjHkWw=";
+  };
+
+  build-system =
+    (with python3Packages; [
+      hatch-vcs
+      hatchling
+      pip
+    ])
+    ++ [ installShellFiles ];
+
+  dependencies = with python3Packages; [
+    jinja2
+    pluggy
+    pyyaml
+    rich
+    requests
+    requirements-parser
+    setuptools
+    tomlkit
+    # Typer needs to be >=0.12.5
+    (typer.overridePythonAttrs (
```

Comment:
I am not sure if an override like this is acceptable in a leaf package.

What I do know is that typer is load bearing for a lot of python utils so I'd expect a version bump like this to blow a lot of packages up.